File: erisbot/src/main/java/studio/styx/erisbot/utils/EmbedReply.java
package studio.styx.erisbot.utils;

import net.dv8tion.jda.api.EmbedBuilder;
import net.dv8tion.jda.api.entities.MessageEmbed;

import java.awt.*;

public class EmbedReply {
    public EmbedBuilder dangerBuilder(String message) {
        return new EmbedBuilder()
                .setDescription(message)
                .setColor(Color.RED);
    }
    public EmbedBuilder successBuilder(String message) {
        return new EmbedBuilder()
                .setDescription(message)
                .setColor(Color.GREEN);
    }
    public EmbedBuilder infoBuilder(String message) {
        return new EmbedBuilder()
                .setDescription(message)
                .setColor(Color.BLUE);
    }

    public MessageEmbed danger(String message) {
        return dangerBuilder(message).build();
    }

    public MessageEmbed success(String message) {
        return successBuilder(message).build();
    }

    public MessageEmbed info(String message) {
        return infoBuilder(message).build();
    }
}

File: common-discord/src/main/kotlin/utils/EmbedReply.kt
package utils

import net.dv8tion.jda.api.EmbedBuilder
import net.dv8tion.jda.api.entities.MessageEmbed
import java.awt.Color

object Embed {
    @JvmStatic
    fun dangerBuilder(message: String?): EmbedBuilder {
        return EmbedBuilder()
            .setDescription(message)
            .setColor(Color.RED)
    }

    @JvmStatic
    fun successBuilder(message: String?): EmbedBuilder {
        return EmbedBuilder()
            .setDescription(message)
            .setColor(Color.GREEN)
    }

    @JvmStatic
    fun infoBuilder(message: String?): EmbedBuilder {
        return EmbedBuilder()
            .setDescription(message)
            .setColor(Color.BLUE)
    }

    @JvmStatic
    fun danger(message: String?): MessageEmbed {
        return dangerBuilder(message).build()
    }

    @JvmStatic
    fun success(message: String?): MessageEmbed {
        return successBuilder(message).build()
    }

    @JvmStatic
    fun info(message: String?): MessageEmbed {
        return infoBuilder(message).build()
    }
}